integration-tests/src/test/java/com/datastax/oss/driver/api/core/ConnectKeyspaceIT.java

```diff
@@ -27,12 +27,16 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
 
 @Category(ParallelizableTests.class)
 public class ConnectKeyspaceIT {
-  @ClassRule public static CcmRule ccm = CcmRule.getInstance();
+  private static CcmRule ccm = CcmRule.getInstance();
```

Comment:
JUnit only accesses the chain, so individual rules don't need to be public anymore.